NuRadioReco/modules/io/noise/noiseImporterUproot.py

```diff
@@ -0,0 +1,195 @@
+import random
+import logging
+import uproot
+import numpy as np
+from scipy import interpolate
+from NuRadioReco.modules.base.module import register_run
+from NuRadioReco.utilities import units
+logger = logging.getLogger('noiseImporter')
+
+# layout to read the ARIANNA data for unknown formatting uproot does not guess right
+ARIANNA_uproot_interpretation = {
+        # TTimeStamp objects in root are 2 ints
+        # * first is time in sec since 01/01/1970,
+        # * second one is nanoseconds
+        # will return a jagged array of shape n_events x [t_s, t_ns]
+        "time": uproot.interpretation.jagged.AsJagged(
+            uproot.interpretation.numerical.AsDtype('>i4'), header_bytes=6),
+        # Interpretation of the trigger mask.
+        # - Bit 0: Thermal trigger
+        # - Bit 1: Forced trigger
+        # - Bit 2: External trigger (not used with current DAQ)
+        # - Bit 3: L1 Trigger satisfied: the L1 trigger cuts away events with
+        #          a large fraction of power in a single frequency.
+        #          true = event PASSES, false = event would be cut by this L1
+        #          (may be in data via scaledown)
+        # - Bit 4: 0 <not used>
+        # - Bit 5: whether event is written thanks to L1 scaledown
+        # - Bit 6: whether throwing away events based on L1 triggers
+        # - Bit 7: flag events that took too long to get data from dCards to MB
+        # NB: No idea why the number of header_bytes in the root files is so odd.
+        "trigger": uproot.interpretation.jagged.AsJagged(
+            uproot.interpretation.numerical.AsDtype("uint8"), header_bytes=7)
+        }
+
+# arianna tigger map
+ARIANNA_TRIGGER = {
+        "thermal" : 2**0,
+        "forced"  : 2**1,
+        "external": 2**2,
+        "l1":       2**3,
+        # not_used: 2**4,
+        "l1_scaledown":     2**5,
+        "l1_enabled" :      2**6,
+        "exceeding_buffer": 2**7
+        }
+
+class noiseImporter:
+    """
+    Imports recorded noise from ARIANNA station.
```

Comment:
Should we rename this to ARIANNA noise importer or something? Otherwise, it seems to be generic and I don't assume it works with RNO-G data for example.